wiki/markdown/running-remrem-publish.md

```diff
@@ -4,18 +4,28 @@ RemRem-Publish can be run from source code with maven command and with RemRem-Pu
 
 ## Maven
 
+### Running RemRem-Publish service
+
 To run RemRem-Publish from source code root directory with maven command and RemRem-Publish Spring/Java properties provided on command line:
 
-    mvn spring-boot:run -Dlogging.level.root=DEBUG -Dgenerate.server.uri=http://127.0.0.1:8080
+    mvn -pl publish-service -am spring-boot:run -Dlogging.level.root=DEBUG -Dgenerate.server.uri=http://127.0.0.1:8080
```

Comment:
can we also mention how to run it from the service itself without having to go for root directory?

Reply:
How do you mean?  You have example how to execute with with Java command and in Tomcat further down on this page.
mvn spring-boot:run works only from root directory, due to how pom files structure is in RemRem-Publish and all version properties/variables in defined in root dir pom file.

Reply:
ok, if it doesn't run from the individual projects then i guess this looks good.

Reply:
If you replace the variables with real version numbers, not variables, in e.g. publish-service/pom.xml,, then it works to execute mvn spring-boot:run in the publish-service folder. So its only possible to execute from root dir as described above.